### PR TITLE
fix: Activity event improvements

### LINKED ISSUE #5775: Activity event improvements
the events in activity are not consistent enough and sometimes a bit broken in term of layout
addition for: 
- #5750

### Mobile and Tablet
image
- Main change is going to be here
- events are going to get bigger and will have more space, bigger preview 
image
- updated in figma - collection detail handoff: https://www.figma.com/file/7BWdGiowrvTucpSFLE0kRX/Collection-detail-handoff?node-id=0%3A1&t=jjaSaxxtfmbYWNc5-1
- if address from either "from" or "to" is empty, the whole event is removed, like with mint 
image
- this does not happen with price, if there is not price, -- is there, this will provide some balance
- preview is now bigger - 70px
- please make sure the preview is square all the time and check if the preview does not stretch the imgs, but scale them
- check exact padding in figma please 

### Changes on desktop
- overflowing text to second row is an issue
image
- text should be cut after a certain length to ensure it does not jump to second row - we can add 3 dots and tooltip on hover - #5704 
- the row could have also better flexible padding between the columns - for example event column has always the same width, so we can maybe lower the margin between price and event etc. 

Let's see how this turns out.
Thanks!

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5775
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

image

### <samp>🤖 Generated by Copilot at e83d93e</samp>

Refactored and improved the layout and responsiveness of the collection activity event components. Used props, utility classes, and flexbox to control the size, alignment, and overflow of the avatar, event, and text elements. Modified `EventRowAvatar.vue`, `EventRowDesktop.vue`, `EventRowTablet.vue`, and `Events.vue`.

### <samp>🤖 Generated by Copilot at e83d93e</samp>

> _Sing, O Muse, of the skillful coder who refines_
> _The `EventRowAvatar` component with a `size` prop divine_
> _And who enhances the event row for desktop view sublime_
> _With utility classes and overflow control, a master of design_

### CODE REVIEW

**Comment by exezbcz** [comment]
hey hey, no jumping to second row please
image

**Review by daiagi — CHANGES_REQUESTED** [review]
> please make sure the preview is square all the time and check if the preview does not stretch the imgs, but scale them

I think ```EventRowAvatar``` is missing ```object-fit: cover;```

* I think ```EventRowAvatar``` can be made into ```NeoAvatar``` component
and can be used in Activity, MassMint, notifications, more?

File: components/collection/activity/events/Events.vue [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <template>
   <div ref="eventsContainer">
     <div v-if="events.length > 0">
-      <div v-if="desktop" class="columns is-size-7 has-text-grey">
+      <div v-if="desktop" class="columns is-1 is-size-7 has-text-grey">
         <div class="column">
           <span>{{ $t('activity.event.item') }}</span>
         </div>
-        <div class="column">
+        <div class="is-flex is-align-items-center fixed-width-100px">
```
Comment (daiagi) [inline components/collection/activity/events/Events.vue:8]: did ```column is-1``` didn't work for you?

the ```is-1``` above in line 4 doesn't do anything

Comment (Jarsen136) [reply components/collection/activity/events/Events.vue]: Thanks, it works.

**Comment by Jarsen136** [comment]
> hey hey, no jumping to second row please image

Fixed

**Review by daiagi — APPROVED** [review]

**Comment by Jarsen136** [comment]
> hmm... not sure about that `NeoAvatar` this name would refer to Identity Avatar to me. I would keep it outside `libs/ui` for now
> 
IMO, this component could also be used for displaying the Identity avatar if we give it proper props. So, let's leave it here for now ?


> What if we want to use lazy loading on these images?

Then we need to introduce `nuxt-img` or some other lazy load package. There is already an issue for `nuxt-image`.

**Comment by Jarsen136** [comment]
> 
> > Then we need to introduce `nuxt-img` or some other lazy load package. There is already an issue for `nuxt-image`.
> 
> fyi `libs/ui` isn't a nuxt project (vue 3)

Thanks for correcting me. Then it would need another lazy load package supported vue3.

**Comment by vikiival** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 30.55 USD/KSM ~ 1.637 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xe56f6966f8b7e589316594e113eb2e189e3805b6bdcd83feef7ac9a72158c638](https://kusama.subscan.io/extrinsic/0xe56f6966f8b7e589316594e113eb2e189e3805b6bdcd83feef7ac9a72158c638)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**